docs/en-US/component/config-provider.md

```diff
@@ -37,34 +37,36 @@ config-provider/message
+| locale                | Locale Object                                                                                                                                                          | ^[object]`{name: string, el: TranslatePair}`[](https://github.com/element-plus/element-plus/blob/a98ff9b40c0c3d2b9959f99919bd8363e3e3c25a/packages/locale/index.ts#L5) [languages](https://github.com/element-plus/element-plus/tree/dev/packages/locale/lang) | [en](https://github.com/element-plus/element-plus/blob/dev/packages/locale/lang/en.ts) |
+| size                  | global component size                                                                                                                                                  | ^[enum]`'large' \| 'default' \| 'small'`                                                                                                                                                                                                                       | default                                                                                |
+| zIndex                | global Initial zIndex                                                                                                                                                  | ^[number]                                                                                                                                                                                                                                                      | —                                                                                      |
+| namespace             | global component className prefix (cooperated with [$namespace](https://github.com/element-plus/element-plus/blob/dev/packages/theme-chalk/src/mixins/config.scss#L1)) | ^[string]                                                                                                                                                                                                                                                      | el                                                                                     |
+| button                | button related configuration, [see the following table](#button-attributes)                                                                                            | ^[object]`{autoInsertSpace?: boolean}`                                                                                                                                                                                                                         | see the following table                                                                |
```

Comment:
I found that the type here is detailed in the Button Attribute table below, does the definition here seem repetitive? Is it better to use the previous `ButtonGlobalConfig `? 🤔

<img width="844" alt="image" src="https://user-images.githubusercontent.com/23100055/205417848-3a8b0789-0be8-4dd9-ae1e-14e3f8a460d3.png">

Reply:
Or can we directly remove the attribute description below and change it to link to button attributes， WDYT?

Reply:
I think the idea is good, but for `message`, there seems to be nowhere to link.

<img width="871" alt="image" src="https://user-images.githubusercontent.com/23100055/205431821-7ad2c85c-f512-423a-a841-40405caa41e5.png">

Reply:
It seems that the corresponding attribute description cannot be canceled. This is a global attribute, and perhaps it can only be kept like this. The above tooltip is just for quick viewing of attributes.

Reply:
Well, it seems so.